src/OctoshiftCLI.Tests/gei/Services/GhesVersionCheckerServiceTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using OctoshiftCLI.GithubEnterpriseImporter.Services;
+using Xunit;
+
+namespace OctoshiftCLI.Tests.GithubEnterpriseImporter.Services
+{
```

Comment:
nit: convert to file scoped namespace to reduce indentation.

Reply:
done https://github.com/github/gh-gei/pull/924/commits/e39386d888d7451ff3c747fe3a1d0badd7a73c4d